source/elements/oneDAL/source/algorithms/decomposition/pca.rst

```diff
@@ -32,7 +32,31 @@ oneDAL specifies two methods for PCA computation:
 -----------------------
 Covariance-based method
 -----------------------
-[TBD]
+
+This method uses eigendecomposition of the covariance matrix to find the
+principal components of the dataset. The following steps are distinguished:
+
+#. Covariance computation
+#. Eigenvectors and eigenvalues computation
+#. Resulting matrix formation
+
+Covariance matrix computation shall be performed in the following way:
+
+#. Compute the sums :math:`s_i = \sum_{j=1}^n x_{i,j}, \quad 1 \leq i \leq p`.
+
+#. Compute the cross-product :math:`P = XX^T - s^Ts`.
```

Comment:
X^T X ?
s-vector-column or vector-row?- explicitly specify and adjust the formula, if necessary

Reply:
changed description of `s` above.